definitions/pipelines/Chipseq_alignment.cwl

```diff
@@ -0,0 +1,150 @@
+requirements:
+    - class: SchemaDefRequirement
+      types:
+          - $import: ../types/labelled_file.yml
+    - class: SubworkflowFeatureRequirement
+inputs:
+    reference: string
+    bams:
+        type: File[]
+    readgroups:
+        type: string[]
+    mills:
+        type: File
+        secondaryFiles: [.tbi]
+    known_indels:
+        type: File
+        secondaryFiles: [.tbi]
+    dbsnp_vcf:
+        type: File
+        secondaryFiles: [.tbi]
+    omni_vcf:
+        type: File
+        secondaryFiles: [.tbi]
+    intervals:
+        type: File
+    per_base_intervals:
+        type: ../types/labelled_file.yml#labelled_file[]
+        default: []
+    per_target_intervals:
+        type: ../types/labelled_file.yml#labelled_file[]
+        default: []
+    summary_intervals:
+        type: ../types/labelled_file.yml#labelled_file[]
+        default: []
+    picard_metric_accumulation_level:
+        type: string
+    bqsr_intervals:
+        type: string[]?
+    minimum_mapping_quality:
+        type: int?
+    minimum_base_quality:
+        type: int?
+    tag_directory_name:
+        type: string?
+        default: "tag_directory"
+outputs:
+    bam:
+        type: File
+        outputSource: alignment/final_bam
+    mark_duplicates_metrics:
+        type: File
+        outputSource: alignment/mark_duplicates_metrics_file
+    insert_size_metrics:
+        type: File
+        outputSource: qc/insert_size_metrics
+    insert_size_histogram:
+        type: File
+        outputSource: qc/insert_size_histogram
+    alignment_summary_metrics:
+        type: File
+        outputSource: qc/alignment_summary_metrics
+    gc_bias_metrics:
+        type: File
+        outputSource: qc/gc_bias_metrics
+    gc_bias_metrics_chart:
+        type: File
+        outputSource: qc/gc_bias_metrics_chart
+    gc_bias_metrics_summary:
+        type: File
+        outputSource: qc/gc_bias_metrics_summary
+    wgs_metrics:
+        type: File
+        outputSource: qc/wgs_metrics
+    flagstats:
+        type: File
+        outputSource: qc/flagstats
+    verify_bam_id_metrics:
+        type: File
+        outputSource: qc/verify_bam_id_metrics
+    verify_bam_id_depth:
+        type: File
+        outputSource: qc/verify_bam_id_depth
+    per_base_coverage_metrics:
+        type: File[]
+        outputSource: qc/per_base_coverage_metrics
+    per_base_hs_metrics:
+        type: File[]
+        outputSource: qc/per_base_hs_metrics
+    per_target_coverage_metrics:
+        type: File[]
+        outputSource: qc/per_target_coverage_metrics
+    per_target_hs_metrics:
+        type: File[]
+        outputSource: qc/per_target_hs_metrics
+    summary_hs_metrics:
+        type: File[]
+        outputSource: qc/summary_hs_metrics
+    bamcoverage_bigwig:
+        type: File
+        outputSource: qc/bamcoverage_bigwig
+    tag_directory:
+        type: Directory
+        outputSource: homer_tag_directory/tag_directory
```

Comment:
Is there any reason to make this dir name configurable? In the original bash script, it was because I was running multiple samples in the same directory, but in this case, every build is going to have it's own dir.  It doesn't hurt anything, I don't think, but might add some unnecessary complexity